client/v2/client.go

```diff
@@ -61,6 +70,9 @@ type Client interface {
 	// Query makes an InfluxDB Query on the database. This will fail if using
 	// the UDP client.
 	Query(q Query) (*Response, error)
+
+	// Close releases any resources a Client may be using.
+	Close() error
```

Comment:
In this case I think `Open` is better. Is a UDPClient opened if `err` is non-nil on its instantiation? Is it OK to call `Close` on a non-nil UDP client? Etc, etc.

Thoughts?